### DISCUSSION TITLE
Registro

### DISCUSSION THREAD

**Discussion by luigirotondi2009** [iisgiua/giuaschool#664] [Richieste di aiuto]
Salve, potete aiutarmi ad installare giua@school?
ho già un sito altervista ma non so come si inserisce la documentazione.
Perfavore aiutatemi

**Comment by trinko** [comment]
A quale documentazione ti riferisci?

**Reply by luigirotondi2009** [reply]
Per installare giua@school

**Comment by trinko** [comment]
[Installazione](https://iisgiua.github.io/giuaschool-docs/install-zip.html)

**Reply by luigirotondi2009** [reply]
![2024-03-12_07-27.log.txt](https://github.com/iisgiua/giuaschool/files/14568144/2024-03-12_07-27.log.txt)

Non mi fa procedere al cambio d'anno.
File in alto.

**Comment by trinko** [comment]
> ![2024-03-12_07-27.log.txt](https://github.com/iisgiua/giuaschool/files/14568144/2024-03-12_07-27.log.txt)
> 
> Non mi fa procedere al cambio d'anno. File in alto.

[Vedi questa segnalazione](https://github.com/iisgiua/giuaschool/issues/655#issuecomment-1962088813)
